zipline/finance/risk/period.py

```diff
@@ -0,0 +1,200 @@
+#
+# Copyright 2018 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logbook
+
+from six import iteritems
+
+import numpy as np
+import pandas as pd
+
+from empyrical import (
+    alpha_beta_aligned,
+    annual_volatility,
+    cum_returns,
+    downside_risk,
+    max_drawdown,
+    sharpe_ratio,
+    sortino_ratio
+)
+
+log = logbook.Logger('Risk Period')
+
+
+# check if a field in rval is nan, and replace it with
+# None.
+def check_entry(key, value):
+    if key != 'period_label':
+        return np.isnan(value) or np.isinf(value)
+    else:
+        return False
+
+
+class RiskMetricsPeriod(object):
```

Comment:
This whole class is actually just a function. The constructor triggers a call to `calculate_metrics` but we don't actually manage state. Can we just make this a function in the report file?